hvplot/converter.py

```diff
@@ -1791,6 +1802,8 @@ def method_wrapper(ds, x, y):
                 style['cmap'] = self._style_opts['cmap']
             if self._dim_ranges.get('c', (None, None)) != (None, None):
                 style['clim'] = self._dim_ranges['c']
+            if self.pixel_ratio:
```

Comment:
Oh I think `datashade` should also support `pixel_ratio`.

Reply:
Done!